Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/kubectl.go

```diff
@@ -165,6 +165,19 @@ var shortForms = map[string]string{
 	"svc":    "services",
 }
 
+// Look-up for resource short forms by value
```

Comment:
For documenting functions, the commentary starts with the function name, and goes on as a normal sentence in English, focused on explaining to the user of the function what it is for and how to use it, etc, and end in a full stop.

That's how we get awesome generated documentation like you see in https://docs.golang.org, or even https://godoc.org/github.com/kubernetes/kubernetes (not so great).

Example:

```go
// ResourceShortFormFor returns the short form of a resource name and whether it
// exists. It returns the empty string if there is no short form.
func ResourceShortFormFor(resource string) (string, bool) {
```

Writing the documentation above made me think about using that function, and that maybe it's signature and contract is not as useful as it could be.

Every time you use it, you need to check if the last argument was true, and only then replace some variable with the alias...

What we have now:

```go
			if alias, ok := kubectl.ResourceShortFormFor(mapping.Resource); ok {
				resourceName = alias
			} else if resourceName == "" {
				resourceName = "none"
 			}
```

Alternative:

```go
			resourceName = kubectl.ShortenResourceName(resourceName)
			if resourceName == "" {
				resourceName = "none"
 			}
// ...

// ShortenResourceName returns the short form of a resource name if there is
// any, otherwise returns the name unchanged.
func ShortenResourceName(name string) string {
	for short, resource := range shortForms {
		if name == resource {
			return short
		}
	}
	return name
}
```